app/controllers/open_studios_subdomain/artists_controller.rb

```diff
@@ -0,0 +1,13 @@
+module OpenStudiosSubdomain
+  class ArtistsController < BaseOpenStudiosController
+    def show
+      (redirect_to '/error' and return) unless FeatureFlags.virtual_open_studios?
+
+      artist = Artist.active.joins(:open_studios_events).friendly.find(params[:id])
+      @artist = ArtistPresenter.new(artist) if artist&.doing_open_studios?
```

Comment:
We may not need this guard here.  I wonder if we *do* need it, what happens if artist.doing_open_studios is false?  I think the join will get you through here safely without it.